build-system/conformance-config.textproto

```diff
@@ -168,6 +168,22 @@ requirement: {
 }
 
 requirement: {
+  type: BANNED_PROPERTY_READ
```

Comment:
No, why that one in particular?
In general we need it when the property contains externally defineable (non-standard) complex data structures.